index.css

```diff
@@ -0,0 +1,58 @@
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  background-color: #bebebe;
```

Comment:
1. consider giving width and height to 100% here.
2. font family missing.
3. no default font size given.

Comments repeated from prev assignment.